mlir/python/pyproject.toml

```diff
@@ -0,0 +1,40 @@
+# Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+# Copyright (c) 2025.
+
+[project]
+name = "mlir-python-bindings"
+version = "0.0.1"
+requires-python = ">=3.8,<=3.13"
+[project.urls]
+Homepage = "https://github.com/llvm/llvm-project"
```

Comment:
Some additional URLs could be added, maybe these two:
"Issue Tracker" = "https://github.com/llvm/llvm-project/issues?q=is%3Aissue%20state%3Aopen%20label%3Amlir%3Apython%20"
"Source Code" = "https://github.com/llvm/llvm-project/tree/main/mlir/python"